cn-india/middlewares/isAuth.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const User = require("../models/user");
+var jwt = require("jsonwebtoken");
+
+module.exports = async (req, res,next) => {
+  const authorization = req.get("Authorization");
+  const token = authorization.split(' ')[1];
+  // console.log(token)
+ 
+  const decode_token = jwt.decode(token, "mysecretkey");
```

Comment:
please store secret key in .env file and then use it here using process.env.KEY_NAME

Reply:
jwt.decode is a wrong method, you should use jwt.verify method